plugin/pkg/admission/limitranger/admission.go

```diff
@@ -163,50 +152,19 @@ func (l *LimitRanger) GetLimitRanges(a admission.Attributes) ([]*corev1.LimitRan
 		return nil, admission.NewForbidden(a, fmt.Errorf("unable to %s %v at this time because there was an error enforcing limit ranges", a.GetOperation(), a.GetResource()))
 	}
 
-	// if there are no items held in our indexer, check our live-lookup LRU, if that misses, do the live lookup to prime it.
-	if len(items) == 0 {
```

Comment:
scope this PR to just the resourcequota changes... dropping the live lookup on limitranger has actual user-visible functional implications during rapid initialization I'm not as sure about, let's move that to a distinct PR to think through

Reply:
Done.